mcs/build/tests.make

```diff
@@ -141,10 +143,49 @@ ifdef TEST_NUNITLITE_APP_CONFIG_RUNTIME
 	sed -i -e "/__INSERT_CUSTOM_APP_CONFIG_RUNTIME__/r $(TEST_NUNITLITE_APP_CONFIG_RUNTIME)" $(NUNITLITE_CONFIG_FILE)
 endif
 
+ifdef PLATFORM_AOT_SUFFIX
+
+DEDUP_DUMMY_CS=$(topdir)/class/lib/$(PROFILE)/DummyInflated.cs
+DEDUP_DUMMY=$(topdir)/class/lib/$(PROFILE)/DummyInflated.dll
+
+$(DEDUP_DUMMY):
+	echo " // Empty Assembly \n\n" > $(DEDUP_DUMMY_CS)
+	$(CSCOMPILE) -t:library -out:$(DEDUP_DUMMY) $(DEDUP_DUMMY_CS) 
+	rm $(DEDUP_DUMMY_CS)
+
+MKBUNDLE_TEST_BIN = $(TEST_HARNESS).static
+MKBUNDLE_EXE = $(topdir)/class/lib/$(PROFILE)/mkbundle.exe
+TEST_ASSEMBLIES := $(sort $(patsubst .//%,%,$(filter-out %.exe.static %.dll.dll %.exe.dll %bare% %plaincore% %secxml% %Facades% %ilasm%,$(filter %test.dll nunit%.dll nunit%.exe,$(wildcard $(topdir)/class/lib/$(PROFILE)/*)))))
```

Comment:
I agree.

Question: how hard do you think would it be instead of compiling everything into the `class/lib/$(PROFILE)` folder to compile/copy all the assets into a separate directory e.g. `class/lib/test` and run everything from there?

This would have the benefit of not polluting the profile output dir, being more similar to how the products use it and also something I'd like to do longer-term anyway for the other profiles (with Jenkins pipeline we could e.g. zip up the directory, put it on another machine and run suites in parallel across machines).